README.md

```diff
@@ -21,24 +21,54 @@ $ python setup.py install # install py-common
 
 ## Usage
 
-**Note:** Before using the py-common in your operating system, you need to make a two-step preparation:
+**Note:** Before using the py-common in your application, you need to make the following preparations:
 
-1. Build executables with sdk-go-common cryption tools. To build these tools, you may need to install **golang** package **sdk-go-common**. For more details please refer to [sdk-go-common](https://github.com/arxanchain/sdk-go-common/tree/master/crypto/tools/README.md)
+### 1.Configure your encryption and signing libraries
 
-2. Copy executables **crypto-util** and **sign-util** into your py-common installation path `cryption/utils`.
+1. Build executables with sdk-go-common encryption tools. To build these executables, you need to install **golang** and download **sdk-go-common**. For more details please refer to [sdk-go-common](https://github.com/arxanchain/sdk-go-common/tree/master/crypto/tools/README.md)
 
-If you have no idea where your py-common is installed, use the following command to check out.
+2. Copy the executables **crypto-util** and **sign-util** into your py-common installation path `cryption/utils`
+
+If you have no idea where your py-common is installed, use the following command to check out(you need to leave the py-common code repo before running this command).
```

Comment:
to check it out (...)

the opening parentheses should go one blank after the previous word.

Reply:
repo means repository ?